url.bs

```diff
@@ -2853,7 +2853,7 @@ handled with care to prevent spoofing:
 <h2 id="application/x-www-form-urlencoded"><code>application/x-www-form-urlencoded</code></h2>
 
 <p>The <dfn export id=concept-urlencoded><code>application/x-www-form-urlencoded</code></dfn> format
-provides a way to encode name-value pairs.
+provides a way to encode name-value <a for=/>tuples</a>.
```

Comment:
Since this is a general description, "name-value pair" works, if we take "pair" as the common term rather than the now-removed infra definition. Otherwise, it should probably be "tuples consisting of a name and a value" as below.